technical_masterplan.md

```diff
@@ -218,6 +218,49 @@ This component vocabulary section aims to briefly define the components involved
+
+Broadly stated, voice has evolved over the past decade from call center-centric recognition to a consumer-owned, platform-enabled all-purpose answer machine. In its next phase – one that is unfolding now – it will become the enabler of contextually-aware dialogue and true personalized assistance.
+The next step in the voice standards journey is address the transitions noted above, as well as these more recent evolutions: • Rapid advances in the AI-enabled voice ecosystem and value chain. A unique industry is now being created with independent voice assistance by world-leading technology firms. Voice is increasingly the entry point of a data-consuming engine of technology and economic growth. Today’s voice questions are different, and much larger than before.
+
+• Ongoing advances in AI-enabled voice technology. Several of the standards listed below were developed to enable interactive voice response systems for telephone call centers. We now stand at the edge of contextually-aware voice functionality that enables human-like dialogue.
+
+• The emergence of platform-independent voice assistants. This calls into question platform and assistant interoperability, in which an enterprise-owned independent assistant can speak with all others, regardless of platform or channel. It suggests a standardization of components, the building blocks of voice assistants – components that have emerged from many of the standards listed below.
+
+As the OVN begins its work, it does so with the desire to learn from and build upon a significant number of widely-adopted and currently-relevant standards for voice assistance. The list below identifies existing voice standards that may guide the functionality and definition of voice assistant components (Dialogue Manager, TTS, ASR, NLU, and many others) and governance protocols (a potential destination registry) now in development by the OVN Technical Committee. It is acknowledged that this list may not be comprehensive.
+
+## Existing (Q4 2020) Standards That May Have Relevance to the Work of the OVN 
+
+This list was assembled by Open Voice Network supporters and advisors Dr. Deborah Dahl, Dr. Jim Larson, David Attwater, and Jonathan Eisenzopf. We are deeply grateful for their contributions and ongoing guidance.
+
+Component Potentially related standards
+IPA Client • (X)HTML, specifies visual content and presentation of web pages IPA Service none Dialog Management • Voice Extensible Markup Language (VoiceXML) 2.1 designed for creating audio dialogs that feature synthesized speech, digitized audio, recognition of spoken and DTMF key input, recording of spoken input, telephony, and mixed initiative conversations. Its major goal is to bring the advantages of Web-based development and content delivery to interactive voice response applications. • State Chart XML (SCXML) provides a generic state-machine based execution environment based on CCXML and Harel State Tables. May be used to specify dialogs
```

Comment:
I think some of the formatting was just copy paste from another format; for example the `•` will not render as it is not markdown. Change it to use `-` for the bullets and add a newline for each (you can see it's one be line with no breaks right now)